src/main/java/de/doubleslash/keeptime/common/SvgNodeProvider.java

```diff
@@ -0,0 +1,50 @@
+package de.doubleslash.keeptime.common;
+
+import javafx.geometry.Bounds;
+import javafx.scene.Node;
+import javafx.scene.shape.SVGPath;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.io.InputStream;
+
+public class SvgNodeProvider {
```

Comment:
you may add a short JavaDoc comment to the class explaining the need for it: To load svgs, but as they are not supported out of the box you extract the path from it.